docs/DeveloperGuide.md

```diff
@@ -23,7 +23,7 @@ title: Developer Guide
 
 --------------------------------------------------------------------------------------------------------------------
 
-## **Setting up, getting started**
+## **Setting up and getting started**
```

Comment:
As mentioned in a previous comment, this should be capitalised. Just change the capitalization of the setting up/getting started page.